packages/angular_devkit/build_angular/test/tslint/works_spec_large.ts

```diff
@@ -215,6 +216,37 @@ describe('Tslint Target', () => {
     ).toPromise().then(done, done.fail);
   }, 30000);
 
+  it('works when a file is only part of one project when using two program', (done) => {
+    const overrides: Partial<TslintBuilderOptions> = {
+      tsConfig: ['src/tsconfig.app.json', 'src/tsconfig.spec.json'],
+      files: ['src/foo/foo.component.ts'],
+    };
+
+    host.writeMultipleFiles({ 'src/foo/foo.component.ts': `const foo = '';\n` });
+
+    runTargetSpec(host, tslintTargetSpec, overrides).pipe(
+      tap((buildEvent) => expect(buildEvent.success).toBe(true)),
+    ).toPromise().then(done, done.fail);
+  }, 30000);
+
+  it('errors when file is part of any typescript program', (done) => {
```

Comment:
nit: errors when file is **not** part of any typescript program

Reply:
done, thanks